dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
Maybe I got this all wrong, but what I get from this is : 

1. We need a sqrt(362) to accomodate the old hardcoded regularisation sphere.
2. Instead of changing that, users can play with $lambda'$, which kinda takes into account that hardcoded value.
3. We could change sqrt(362) to actually be the number of point on the regularisation sphere, also defaulting to 362 for the default code path, but also taking into account any other sampled sphere. Granted, anyone using another sphere will get a deffiretn value from now on, but that's correcting a hardcoded value at the same time, which is more sane in the long run.